gatling-http/src/main/scala/io/gatling/http/action/ws/fsm/WsIdleState.scala

```diff
@@ -82,6 +86,9 @@ final class WsIdleState(fsm: WsFsm, session: Session, webSocket: WebSocket, prot
     // actually send message!
     val now = clock.nowMillis
     webSocket.sendFrame(new BinaryWebSocketFrame(Unpooled.wrappedBuffer(message)))
+    val requestMessage =
+      if (HttpTracing.IS_HTTP_TRACE_ENABLED || HttpTracing.IS_HTTP_DEBUG_ENABLED) Some(s"<<<BINARY CONTENT length=${message.length}>>>") else None
```

Comment:
redundant: if trace is enabled, debug is => `if (HttpTracing.IS_HTTP_DEBUG_ENABLED)` is enough.